src/components/chip/chip.classnames.ts

```diff
@@ -0,0 +1,70 @@
+import { ChipSize } from './types'
+import { cva } from 'class-variance-authority'
+
+export const chipBaseVariant = cva('chipBaseClassnames', {
```

Comment:
`chipBaseClassnames` is a list of base class referred from below. Should it be provided as a string for the first prop of `cva`?

Reply:
@likhith-deriv I couldn't get this `It seem to work, but breaks if there are any pseudo selectors in the baseClass`, can you please help me understand this? If you meant  `:hover`, `:active`, etc pseudo selectors behavior then it shouldn't be an issue as we are not using them on this `chipBaseClassnames`. It is only being used for the size variant and the pseudo selectors are being used on other className that are added separately to the component. But please let me know if you meant something else that could be an issue.

Reply:
No. I meant the `:hover` and other pseudos. Thanks for the clarification